packages/api/src/cli/storage.ts

```diff
@@ -0,0 +1,297 @@
+import type { OASDocument } from 'oas/@types/rmoas.types';
```

Comment:
A lot of this is duped from the `src/cache` library that powers the dynamic, non-installed `api`. I'd love to merge the two so if you're using a dynamic `api` you already have your spec installed in `.api` and can easily create an SDK for it, but that'd require us to know the identifier for the spec upfront and we don't have a way to ask these users that without overcomplicating the dynamic `api` API.